docs/cBioPortal-ER-Diagram.md

```diff
@@ -1,2 +1,4 @@
+# cBioPortal ER Diagram
```

Comment:
It's the first link under 4. Development, right?

I did notice now there is some collapse/expand + sign on the sidebar to the left that doesn't behave entirely as expected. Perhaps not a big deal tho